pages/agent/v3/aws.md.erb

```diff
@@ -1,19 +1,15 @@
 # Running Buildkite Agent on AWS
 
-The Buildkite Agent can be run on AWS using our Elastic CI Stack for AWS.
+The Buildkite Agent can be run on AWS using our Elastic CI Stack for AWS
+CloudFormation template, or by installing the agent on your self-managed
```

Comment:
I don't know how much we stress, or want to stress, that this is a CF template. @harrietgrace do you have a read on this?